src/bundle.js

```diff
@@ -44,10 +44,6 @@ module.exports = function bundle(options, bundleModules) {
       if (code !== 0) {
         const errorMessage = stderr.join('');
 
-        if (errorMessage.length) {
```

Comment:
I was going to make a PR after filing #110, but found this PR. I'm not sure that removing the error reporting is the right solution, though.